docs/user-script-execution.md

```diff
@@ -0,0 +1,28 @@
+# User scripts and their execution
+
+## Scripts
+
+In algob, scripts are JS files stored in the `scripts` directory.
+They are used to manage migrations and deployments.
+Child directories are also possible.
+
+Please see the [architecture document](https://paper.dropbox.com/doc/Algorand-builder-architecture--A3aVSVEt3HIRGIiCnTMbn64DAg-Vcdp0XNngizChyUWvFXfs#:uid=213683005476107006060621) to see how the scripts are organized and how to use them.
+
+### Script execution
+
+Scripts can be executed in two ways:
+- `algob run`
+- `algob deploy`
+
+#### Run
+Runs provided scripts by not saving any script state snapshots. Useful to query the current state of blockchain.
```

Comment:
we don't do snapshot. We are doing checkpoints.
Snapshot is a saved state of the system.

Reply:
updated